cobbler/services.py

```diff
@@ -281,7 +283,7 @@ def list(self, what="systems", **rest) -> str:
         else:
             return "?"
         for x in listing:
-            buf += "%s\n" % x["name"]
+            buf += f"{x['name']}\n"
```

Comment:
Strings shouldn't be built in a loop with `+=`, see https://peps.python.org/pep-0008/#programming-recommendations